src/htmlrenderer.cpp

```diff
@@ -642,7 +642,7 @@ void htmlrenderer::render(
 					i+1,
 					links[i].first.c_str(),
 					type2str(links[i].second).c_str());
-			lines.push_back(std::make_pair(newsbeuter::nonwrappable, link_text));
+			lines.push_back(std::make_pair(newsbeuter::softwrappable, link_text));
```

Comment:
This is the same as `add_line_softwrappable`, so just use that. Can you also look through the module and see if there's other places where I forgot to use a function? Thanks.